shells/webextension/build.js

```diff
@@ -21,7 +21,7 @@ const STATIC_FILES = [
   'panel.html',
 ];
 
-const relativePath = path => relative(process.env.PWD, path);
+const relativePath = path => relative(__dirname, path);
```

Comment:
This is relative from `/shells/webextension/build`. Seems confusing.

Can we just use `process.cwd()`?

Reply:
I had forgotten about this PR, but yes, that makes much more sense. Updated.